src/ci/docker/host-x86_64/mingw-check-tidy/Dockerfile

```diff
@@ -1,13 +1,16 @@
 FROM ubuntu:22.04
 
 ARG DEBIAN_FRONTEND=noninteractive
+# NOTE: intentionally uses python2 for x.py so we can test it still works.
+# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
```

Comment:
Can we move this comment next to the `x.py` invocation? Having it 30 lines above the actual usage could lead to the comment being missed.